dashboard/app/models/pd/facilitator_program_registration.rb

```diff
@@ -87,7 +87,13 @@ def self.options
     ).freeze
   end
 
+  def clear_data
+    update!(form_data: {}.to_json)
+  end
```

Comment:
you could more simply define this method on the [shared form concern](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/concerns/pd/form.rb)

Reply:
Done. Much better. :)